src/count_occurrence.md

```diff
@@ -0,0 +1,77 @@
+# count_occurrence
+To Count The Occurrence of value in an array
+
+## Result
+
+```Py
+   count_occurrences([2,2,4,5,6], target_value) // '2'
```

Comment:
This should be the expected result for all snips in the file, instead of every single language